lib/cluster.js

```diff
@@ -285,6 +287,8 @@ function masterInit() {
     var debugPort = process.debugPort + id;
     var hasDebugArg = false;
 
+    debugPort = debugPort % kDebugPortRange + kDebugPortStart;
```

Comment:
Sorry, valid point but consider debugPort == 64511: 64511 % (65536 - 1024) + 1024 = 65536.

Apropos the behavior change, I think node-inspector assumes consecutive ports.